cubedash/summary/_extents.py

```diff
@@ -384,6 +393,7 @@ def _populate_missing_dataset_extents(
                 == bindparam("product_ref", product.id, type_=SmallInteger)
             )
             .where(DATASET.c.archived == None)
+            .where(func.ST_IsValid(DATASET_SPATIAL.c.footprint) != True)
```

Comment:
Not sure about this.

Ideally, the dataset is in Explorer, but the geometry is ignored? Or, we fix geometry at this step?